app/controllers/application_controller.rb

```diff
@@ -78,7 +78,13 @@ def embeddable?
   end
 
   def embedding_without_https?
-    request.referer && URI(request.referer).scheme != 'https' && !Rails.env.test? && !Rails.env.development?
+    request.referer && parsed_referer_uri.scheme != 'https' && !Rails.env.test? && !Rails.env.development?
+  end
+
+  def parsed_referer_uri
+    return @parsed_referer_uri if @parsed_referer_uri
+    referer_url = request.referer.gsub("#", '')
```

Comment:
watch out, Rubocop (in Code Climate) is asking you to use `#delete`.

Reply:
Yes, I should have run rubocop before commiting. Should I make a new commit with the changes required by rubocop ?
I will start doing that on others PR's.

Reply:
It's up to you but normally for these single-commit PRs I just amend the commit where I added the change and I force the git push.

Reply:
> Yes, I should have run rubocop before commiting.

The earlier you get feedback from Rubocop the better. That's why I strongly recommend to [integrate it with your editor](https://rubocop.readthedocs.io/en/latest/integration_with_other_tools/).

Reply:
Thanks, I'll integrate it with Visual Code, I've switched to it recently :+1: 
I've force pushed the changes with the single commit.